<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesViewForm</class>
 <widget class="QWidget" name="SalesViewForm">
  <property name="windowTitle">
   <string>Sales</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_top">
     <item>
      <widget class="QLineEdit" name="txtBarcode">
       <property name="placeholderText">
        <string>Scan or enter barcode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblCart">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_bottom">
     <item>
      <spacer name="horizontalSpacer_bottom">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblTotalAmount">
       <property name="text">
        <string>Total: 0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignVCenter</set>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 18pt; font-weight: bold;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheckout">
       <property name="text">
        <string>Checkout</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>